.github/workflows/release.yaml

```diff
@@ -0,0 +1,58 @@
+name: Release
+
+on:
+  workflow_dispatch: {}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+        fetch-depth: 0
+    - name: Set environment variables
+      run: ./build/ci/set_environment.sh
+    - name: Build chart
+      run: ./build/chart.sh
+      env:
+        IMAGE_REPOSITORY: ghcr.io/${{ github.repository }}
+    - name: Draft release
+      id: draft_release
+      uses: actions/create-release@v1
+      with:
+        release_name: ${{ env.GIT_TAG }}
+        tag_name: ${{ env.GIT_TAG }}
+        body: A MITS release.
+        draft: true
+        prerelease: false
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Upload chart to draft release
+      uses: actions/upload-release-asset@v1
+      with:
+        upload_url: ${{ steps.draft_release.outputs.upload_url }}
+        asset_path: output/mits-${{ env.VERSION }}.tgz
+        asset_name: mits-${{ env.VERSION }}.tgz
+        asset_content_type: application/x-gzip
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Build and publish image
+      uses: docker/build-push-action@v1
+      with:
+        registry: ghcr.io
+        repository: ${{ github.repository }}
+        tags: ${{ env.IMAGE_TAG }}
+        labels: >-
+          org.opencontainers.image.version=${{ env.VERSION }}
```

Comment:
Consider overriding some additional labels that we're inheriting from the base image:

Label | Current Value
 --- | ---
`org.opencontainers.image.description` | Image containing a minimal environment for containers based on openSUSE Leap 15.1.
`org.opencontainers.image.title` | openSUSE Leap 15.1 Base Container
`org.opencontainers.image.url` | https://www.opensuse.org/
`org.opencontainers.image.vendor` | openSUSE Project

I'm ignoring `org.openbuildservice.disturl` and `org.opensuse.*` as irrelevant.

Reply:
Fixed with c7b2cae.